corehq/motech/rate_limiter.py

```diff
@@ -109,8 +110,8 @@ def report_repeater_usage(domain, milliseconds):
 @run_only_when(SHOULD_RATE_LIMIT_REPEATERS)
 @silence_and_report_error("Exception raised reporting usage to the repeater attempt rate limiter",
                           'commcare.repeaters.report_usage_errors')
-def report_repeater_attempt(domain):
-    repeater_attempts_rate_limiter.report_usage(domain)
+def report_repeater_attempt(repeater_id):
+    repeater_attempts_rate_limiter.report_usage(repeater_id)
```

Comment:
I don't see where `report_usage` was changed to accept `repeater_id` instead of `domain`. Does that take a generic value, or was that an oversight?

Reply:
Yeah that is a generic value, effectively a key to save usage values to.